workspaces/cli-shared/src/query/build-query-string-parser.ts

```diff
@@ -0,0 +1,6 @@
+import { IQueryParser } from './query-parser-interfaces';
+import { DefaultQueryParser } from './parsers/query-string/DefaultQueryParser';
+
+export function buildQueryStringParser(): IQueryParser {
+  return new DefaultQueryParser();
```

Comment:
In the future this is the spot where we inspect the user's optic config, determine the selected strategy, and return the appropriate function?

Reply:
yes